templates/data/docs/kafka/iaas/document.html

```diff
@@ -0,0 +1,11 @@
+{% with
```

Comment:
Is this the expected behavior for this one? 
![Screenshot from 2023-07-03 10-38-02](https://github.com/canonical/ubuntu.com/assets/17607612/2aeef1e0-4157-4c12-aa15-7a6b4a6e9e5b)


There's no sidenav and the step by step guide are direct links to discourse, in contrast to the rest of the the tutorials which look like so:
![image](https://github.com/canonical/ubuntu.com/assets/17607612/5f49e26c-76d5-42a3-b667-a98eda1b64be)

Reply:
@deusebio can we add a navigation to this tutorial?

Reply:
Yes. Definitely. Let me follow up on this and give you the heads up once it is fixed

Reply:
The mentioned link: https://ubuntu-com-12969.demos.haus/data/docs//mysql/iaas points to MySQL "Tutorial"
https://discourse.charmhub.io/t/charmed-mysql-tutorial-overview/9922

The correct high-level link is https://discourse.charmhub.io/t/charmed-mysql-documentation/9925 
The correct link used in the charm: https://github.com/canonical/mysql-operator/blob/main/metadata.yaml#L13C7-L13C71

See https://github.com/canonical/ubuntu.com/pull/12969#pullrequestreview-1511173616